.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: golangci-lint
```

Comment:
why are we running `golangci-lint` on release?
I'd imagine we'd only release stuff that has already been checked before merging to `master`.

Reply:
That's a typo, I was copying the YAML spec from lint pipeline. If you read the rest of the file, it just builds the images and pushes them to GHCR